testing/mysql/integration_test.go

```diff
@@ -104,6 +103,7 @@ func prepareIntegrationTest(t *testing.T) string {
 }
 
 func TestIntegration_MYSQLDUMP_SimpleUse(t *testing.T) {
+	onlyRunForEmulatorTest(t)
```

Comment:
Interesting. So now with `if databaseAdmin == nil` check in prepareIntegrationTest() gone, you need to this extra line here since the test still need Spanner emulator to be up and running.

I think our handling for when to skip a given test is kind of fragile and hasn't kept up to date with all the changes that we have made in last 6 months. This is something we should clean up in a follow up PR.